src/lib/sidebar/ListMode.svelte

```diff
@@ -107,7 +101,7 @@
 
 <hr />
 
-{#each Object.keys($gjSchemeCollection.schemes) as scheme_reference, i}
+{#each Object.keys($gjSchemeCollection.schemes) as scheme_reference, i (scheme_reference)}
```

Comment:
There was a major bug here! At the very start, $gjSchemeCollection is just the emptyCollection. This code runs and creates these components, which have an error, since the blank scheme has no name (or other details). We soon load data from local storage. If it had all the details filled out, you'd expect the error message to go away. But very subtly, `PerSchemeControls` wasn't getting recreated for the new scheme(s) loaded. Instead the existing component was just having its `scheme_reference` changed. Keyed each blocks fix. `PerSchemeControls` should only get mounted once per scheme.

Reply:
Great fix by the sounds of it!